.github/workflows/rust.yml

```diff
@@ -268,26 +268,17 @@ jobs:
 
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.48.0
+          toolchain: 1.56.1
           override: true
 
       - name: check if README matches MSRV defined here
-        run: grep '1.48.0' src/core/README.md
-
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v4
-        with:
-          python-version: "3.8"
-
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          python -m pip install -e .
+        run: grep '1.56.1' src/core/README.md
 
       - name: Check if it builds properly
         uses: actions-rs/cargo@v1
         with:
           command: build
+          args: --all-features
```

Comment:
make sure that we track optional features too, so dependabot won't leave us in a broken situation.